### DISCUSSION TITLE
Stuck on Generating SSL Certificate

### DISCUSSION THREAD

**Discussion by LowScarlet** [vercel/community#870] [Help] [answered]
I want to use wildcard domain but stuck on SSL Certificate Generation
![image](https://user-images.githubusercontent.com/64282946/190581506-85150674-83ef-43a7-ac23-f2b0c0ac8cf5.png)

**Comment by swarnava** [comment, marked as answer]
Sorry about this. This should be fixed now: https://www.vercel-status.com/incidents/dcvccs0q7y54

**Reply by Myzel394** [reply]
@swarnava I'm currently facing the same issue :/

**Reply by swarnava** [reply]
Can you create a ticket: https://vercel.com/help?  Thanks!
